### PR TITLE
Issue #614: Resolve CallToSystemExit, string formatting and string builder IDEA violations

### PR DESCRIPTION
**Pull request by Vyom-Yadav**
#614

### CODE REVIEW

File: releasenotes-builder/src/main/java/com/github/checkstyle/Main.java [hunk 1]
```diff
@@ -47,6 +47,8 @@ private Main() {
      * Entry point.
      *
      * @param args command line arguments.
+     * @noinspection CallToSystemExit
+     * @noinspectionreason CallToSystemExit - driver class must call exit
```
Comment (romani) [inline releasenotes-builder/src/main/java/com/github/checkstyle/Main.java:51]: `- main method must exist with code`

**Review by romani — CHANGES_REQUESTED** [review]
items:

File: releasenotes-builder/src/main/java/com/github/checkstyle/Main.java (hunk 1 above)
Comment (Vyom-Yadav) [reply releasenotes-builder/src/main/java/com/github/checkstyle/Main.java]: Done.

**Review by romani — APPROVED** [review]
Ok to merge

**Review by nick-mancuso — APPROVED** [review]

**Comment by nick-mancuso** [comment]
@Vyom-Yadav please resolve conflicts

**Comment by Vyom-Yadav** [comment]
> @Vyom-Yadav please resolve conflicts

Done.

**Pull request merged**